src/scalability/multi-raft.md

```diff
@@ -0,0 +1,17 @@
+# Multi-raft 
+
+If you've researched Consensus before, please note that comparing Multi-Raft to Raft is not at all like comparing Multi-Paxos to Paxos. Here Multi-Raft only means we manage multiple Raft consensus groups on one node. From the above section, we know that there are multiple different partitions on each node, if there is only one Raft group for each node, the partitions losing its meaning. So Raft group is divided into multiple Raft groups in terms of partitions, namely, Region. 
+
+TiKV also can perform split or merge on Regions to make the partitions more flexible. When the size of a Region exceeds the limit, it will be divided into two or more Regions, and the range may change like \\( [a, c) \\) -> \\( [a, b) \\) + \\( [b, c) \\); when the sizes of two sibling Regions is small enough, they will be merged into a bigger Region, and the range may change like \\( [a, b) \\) + \\( [b, c) \\) -> \\( [a, c) \\).
```

Comment:
```suggestion
TiKV also can perform split or merge on Regions to make the partitions more flexible. When the size of a Region exceeds the limit, it will be divided into two or more Regions, and the range may change like \\( [a, c) \\) -> \\( [a, b) \\) + \\( [b, c) \\); when the sizes of two sibling Regions are small enough, they will be merged into a bigger Region, and the range may change like \\( [a, b) \\) + \\( [b, c) \\) -> \\( [a, c) \\).
```